Looks good to me.

@mattclay Can we disable PEP8 until we figure out what we want to test for, and how to fit it in the workflow ? Now it bails out on unrelated code, confusing contributors for no good reason.

```
2017-02-06 15:15:17 ERROR: PEP 8: contrib/inventory/packet_net.py:146:11: E111 indentation is not a multiple of four (current)
2017-02-06 15:15:17 ERROR: PEP 8: contrib/inventory/packet_net.py:149:15: E111 indentation is not a multiple of four (current)
2017-02-06 15:15:17 ERROR: PEP 8: contrib/inventory/packet_net.py:152:11: E111 indentation is not a multiple of four (current)
2017-02-06 15:15:17 ERROR: PEP 8: contrib/inventory/packet_net.py:203:73: E703 statement ends with a semicolon (current)
2017-02-06 15:15:17 ERROR: PEP 8: contrib/inventory/packet_net.py:307:37: E712 comparison to True should be 'if cond is True:' or 'if cond:' (current)
2017-02-06 15:15:17 ERROR: PEP 8: contrib/inventory/packet_net.py:392:30: E721 do not compare types, use 'isinstance()' (current)
```

--- a/lib/ansible/parsing/mod_args.py
+++ b/lib/ansible/parsing/mod_args.py
@@ -287,7 +287,7 @@ class ModuleArgsParser:
             if item in module_loader or item in ['meta', 'include', 'include_role']:
                 # finding more than one module name is a problem
                 if action is not None:
-                    raise AnsibleParserError("conflicting action statements", obj=self._task_ds)
+                    raise AnsibleParserError("conflicting action statements: {}, {}".format(action, item), obj=self._task_ds)
                 action = item
                 thing = value
                 action, args = self._normalize_parameters(thing, action=action, additional_args=additional_args)

--- a/lib/ansible/parsing/mod_args.py
+++ b/lib/ansible/parsing/mod_args.py
@@ -287,7 +287,7 @@ class ModuleArgsParser:
             if item in module_loader or item in ['meta', 'include', 'include_role']:
                 # finding more than one module name is a problem
                 if action is not None:
-                    raise AnsibleParserError("conflicting action statements: {}, {}".format(action, item), obj=self._task_ds)
+                    raise AnsibleParserError("conflicting action statements: %s, %s" % (action, item), obj=self._task_ds)
                 action = item
                 thing = value
                 action, args = self._normalize_parameters(thing, action=action, additional_args=additional_args)

